Code/BuildSystem/CMake/CMakeUtils/ezUtilsCppFlags.cmake

```diff
@@ -240,6 +189,11 @@ function(ez_set_build_flags_clang TARGET_NAME)
 	else()
 		target_compile_options(${TARGET_NAME} PRIVATE "--system-header-prefix=\"${CMAKE_SOURCE_DIR}/Code/ThirdParty\"")
 	endif()
+
+	if (COMMAND ez_platformhook_set_build_flags_clang)
```

Comment:
shouldn't we have something similar for gcc and msvc?

Reply:
Yes, there is still a lot to do.